src/components/victory-axis.jsx

```diff
@@ -507,22 +474,25 @@ export default class VictoryAxis extends React.Component {
       // Do less work by having `VictoryAnimation` tween only values that
       // make sense to tween. In the future, allow customization of animated
       // prop whitelist/blacklist?
-      const animateData = _.omit(this.props, [
-        "orientation", "scale", "tickFormat", "animate",
-        "crossAxis", "standalone"
+      const animateData = _.pick(this.props, [
+        "style", "domain", "range", "tickCount", "tickValues",
+        "labelPadding", "offsetX", "offsetY", "padding", "width", "height"
       ]);
       return (
         <VictoryAnimation {...this.props.animate} data={animateData}>
-          {props => <VictoryAxis {...this.props} {...props} animate={null}/>}
+          {(props) => <VictoryAxis {...this.props} {...props} animate={null}/>}
         </VictoryAnimation>
       );
+    } else {
+      this.getCalculatedValues(this.props);
     }
+    const transform = this.getTransform(this.props);
     const group = (
-      <g style={this.style.parent} transform={this.transform}>
-        {this.getGridLines()}
-        {this.getAxisLine()}
-        {this.getTickLines()}
-        {this.getLabelElements()}
+      <g style={this.style.parent} transform={transform}>
+        {this.renderGridLines()}
+        {this.renderAxisLine()}
+        {this.renderTicks()}
+        {this.renderLabel()}
```

Comment:
This is a convention I think we should adopt, and one I've seen elsewhere: If we break out rendering into smaller methods, those methods should be named like `renderFoo` – that way it's clear when they get called and what they return.